code/demo/android/app/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
 
   lintOptions {
     abortOnError true
-    warningsAsErrors true
+    warningsAsErrors false
```

Comment:
Why the need to allow warnings?

Reply:
My build was failing due to some warnings and I didn't want to change to project setup that caused these warning so I just made them appears as what they really are warnings, not errors (makes the build fail).

Reply:
Then we should fix the warnings.

Reply:
for me this is the latest "issue" that needs to be addressed.
Could you attach the warnings here ?

Reply:
There were 7 errors and failed gradle build for demo/android and 2 lint warnings and NOT failed build for demo/android-native
 
warning: ObsoleteLintCustomCheck: Obsolete custom lint check
Lint found an issue registry (androidx.fragment.lint.FragmentIssueRegistry) which is older than the current API level; these checks may not work correctly.
Recompile the checks against the latest version. Custom check API version is 6 (3.6), current lint API level is 8 (4.1+)

error: GradleDependency: Obsolete Gradle Dependency
../../build.gradle:69: A newer version of androidx.fragment:fragment than 1.2.5 is available: 1.3.1

error: LockedOrientationActivity: Incompatible screenOrientation value
../../src/main/AndroidManifest.xml:13: Expecting android:screenOrientation="unspecified" or "fullSensor" for this activity so the user can use the application in any orientation and provide a great experience on Chrome OS devices

error: AllowBackup: AllowBackup/FullBackupContent Problems
../../src/main/AndroidManifest.xml:4: On SDK version 23 and up, your app data will be automatically backed up and restored on app install. Consider adding the attribute android:fullBackupContent to specify an @xml resource which configures which files to backup. More info: https://developer.android.com/training/backup/autosyncapi.html

error: ObsoleteSdkInt: Obsolete SDK_INT Version Check
../../src/main/res/values-v14: This folder configuration (v14) is unnecessary; minSdkVersion is 17. Merge all the resources in this folder into values.

error: UnusedResources: Unused resources
../../src/main/res/values-v14/themes.xml:3: The resource R.style.MyTheme appears to be unused

error: ClickableViewAccessibility: Accessibility in Custom Views
../../../../../build/android/java/org/orx/lib/OrxActivity.java:85: Custom view `SurfaceView` has setOnTouchListener called on it but does not override performClick
../../../../../build/android/java/org/orx/lib/OrxActivity.java:238: OrxActivity#onTouch should call View#performClick when a click is detected

I'd better just revert this so we see these and fix these.
Currently these can be seen but the build does not fail because if the flag. 
> Task :app:lint
Ran lint on variant debug: 8 issues found
Ran lint on variant release: 7 issues found

Some of these warning I think has to do with the changes to use Fragment by @l33t

Reply:
I've reverted this now so its like it was before.
These lint errors problem for gradle build should be taken care of but not in this PR I think.
I've actually fixed them now but I'll make another PR with the fixes for @znakeeye to review :-)